javascript/javascript2/week1/homeWork/stringsEndsWith.js

```diff
@@ -0,0 +1,8 @@
+function solution(text, ending) {
+  let array = text.split("");
+  array = array.slice();
+  console.log(array);
+  return (stringMEthod = text.endsWith(ending));
```

Comment:
```suggestion
  const stringMethod = text.endsWith(ending);
  return stringMethod;
```
Avoid returning variable allocation; it makes your code confusing. I first thought you tried to compare the two values but forgot a `=`.